src/components/MultiActionButton/MultiActionButton.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from 'react';
+import { useSelect } from 'downshift';
+import { usePopper } from 'react-popper';
+import { bem } from '../../utils/bem/bem';
+import { mergeRefs } from '../../utils/mergeRefs';
+import { PopupPlacement } from '../../constants';
+import { ButtonProps } from '../Buttons';
+import { ListItemProps } from '../List/ListItem';
+import { List } from '../List';
+import styles from './MultiActionButton.scss';
+
+const { elem } = bem('MultiActionButton', styles);
+
+interface Props<V> extends Omit<React.HTMLAttributes<HTMLDivElement>, 'onChange'> {
+    /**
+     *  Popup content to be placed inside List.
+     *  Navigation available only if element has not empty `value` attribute
+     *  and empty/false `disabled` attribute.
+     *  Use {@link ListItem} component.
+     */
```

Comment:
Nice! 
It is good to add this to the story too, so we can see it in storybook. For examples see stories with `info` section.

Reply:
What exactly you want to add to stories?
Now we already have a story which shows available/not available for navigation items

Reply:
I would add the whole thing, as a reminder when we want to use this component 3 month down the line, and we forgot. It is true that it is demonstrated in the story code, but it is nice to spell things out sometime, to understand what makes it tick. And you summed it up so nicely already.

Reply:
aaah, oki! I got it